### DISCUSSION TITLE
Glide server cache lifetime

### DISCUSSION THREAD

**Discussion by dave2084** [awcodes/filament-curator#534] [General]
What is the lifetime of the Glide server cache?  

Is this something we can configure?

**Comment by awcodes** [comment]
To my knowledge it uses the default laravel lottery caching strategy. But you can clear it any time you need with a scheduled command. Here's the docs they provide on caching. https://glide.thephpleague.com/2.0/config/source-and-cache/

**Comment by dave2084** [comment]
Thanks!
